### ISSUE TITLE
Re-instaling GitHub and markbot

### ISSUE THREAD

**Issue by kyramari** [thomasjbradley/markbot#130]
Hey, Thomas. So I wiped my laptop and re-put all my files back on just to make my laptop run better. I downloaded GitHub but it isn't set up with markbot or anything. And doesn't have my files. Is there a way I can get it all back without having to set it all up again from scratch? 

Thanks

**Comment by thomasjbradley** [comment]
Yeah, inside the GitHub app there's an add button, I think it's a `+`, in the top left. There should be a button named something like "Add Local Repository". With this you can navigate to where your files are on your computer and it'll reconnect. But you'll have to do it for each folder you want to reconnect.

**Comment by thomasjbradley** [comment]
Unless you're using the brand new version, then there's a button right on the first screen named "Add a Local Repository".

**Comment by kyramari** [comment]
Ya, its the new version I think. And then, would I add a new repository or my actual webdev files from class?

**Comment by thomasjbradley** [comment]
You're looking to bring back in the previous Web Dev class files into the GitHub app?

**Comment by kyramari** [comment]
I'm not sure. ahah. I just want to make sure it's set up properly for class so that I don't have a bunch of problems in September.

**Comment by thomasjbradley** [comment]
Then, it's probably set up okay. Did you checkout the installation instructions for Web Dev 1?
https://learn-the-web.algonquindesign.ca/courses/web-dev-1/install-all-the-things/

**Comment by kyramari** [comment]
Ok. And not yet, but I'll follow those steps now. Thanks for your help! I just wanted to make sure I didn't mess anything up. :)

**Comment by kyramari** [comment]
Oh! one more thing. Where on your github can I find the latest version of markbot to download. I can't seem to figure out where it is for some reason.

**Comment by thomasjbradley** [comment]
It's actually at the bottom of the page on Learn the Web:
https://learn-the-web.algonquindesign.ca/

**Comment by kyramari** [comment]
oh thanks!
